core/domain/image_validation_services.py

```diff
@@ -37,15 +38,18 @@ def validate_image_and_filename(raw_image, filename):
         ValidationError. Image or filename supplied fails one of the
             validation checks.
     """
-    hundred_kb_in_bytes = 100 * 1024
 
+    hundred_kb_in_bytes = 100 * 1024
     if not raw_image:
         raise utils.ValidationError('No image supplied')
     if len(raw_image) > hundred_kb_in_bytes:
         raise utils.ValidationError(
             'Image exceeds file size limit of 100 KB.')
     allowed_formats = ', '.join(
         list(feconf.ACCEPTED_IMAGE_FORMATS_AND_EXTENSIONS.keys()))
+    # TODO(#14204): Refactor this to use the normalized_payload files.
+    if utils.is_base64_encoded(raw_image):
```

Comment:
Why are we expecting to use payload in service files?

Reply:
Yeah, this comment is not required here.